clients/tabby-chat-panel/src/client.ts

```diff
@@ -229,3 +236,31 @@ export interface ListSymbolItem {
    */
   range: LineRange
 }
+
+/**
+ * Parameters for retrieving Git changes from the client IDE.
+ * Used with {@link ClientApi.getChanges} method.
+ */
+export interface GetChangesParams {
+  /**
+   * The maximum number of tokens to return in the change content.
+   * If not provided, all changes will be returned without token limitation.
+   */
+  limitToken?: number
```

Comment:
```suggestion
  maxChars?: number
```
This parameter is used to limit the token count. However, the client cannot calculate the token count accurately, and approximate token count calculations should not be performed on the client side either.  
I prefer using `maxChars` here as it provides a precise definition.